pkg/upload/js.go

```diff
@@ -150,7 +153,17 @@ func uploadSingleSourceMap(
 		logger.Fatal(err.Error())
 	}
 
-	uploadOptions, err := utils.BuildJsUploadOptions(apiKey, appVersion, bundleUrl, projectRoot, overwrite)
+	url := bundleUrl
+	if baseUrl != "" {
+		relativePath, err := filepath.Rel(outputPath, bundlePath)
+		if err != nil {
+			logger.Warn(fmt.Sprintf("cannot make path relative to create URL, skipping: %v", err))
+			return
+		}
+		url = baseUrl + relativePath
```

Comment:
As discussed, let's make this simpler and just prepend the bundle file name to the base dir - that way it's more obvious to the user how this works and in complicated cases where the structures vary in the build and hosted structure, they can work out what to do.

I'd also suggest that this logic is moved up to the `ProcessJs` function so that at this level it just takes a single file to upload with a URL and doesn't need to know about the `outputPath`.